CIP-9999/README.md

```diff
@@ -0,0 +1,189 @@
+---
+CIP: 9999
+Title: Cardano Problem Statements
+Status: Active
+Category: Meta
+Authors:
+    - Matthias Benkort <matthias.benkort@cardanofoundation.org>
+    - Michael Peyton Jones <michael.peyton-jones@iohk.io>
+Implementors: N/A
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pulls/365
```

Comment:
```suggestion
    - https://github.com/cardano-foundation/CIPs/pull/366
```

It looks like this is the correct link to the pull request for discussion.

Reply:
Indeed, I picked 365 in anticipation but there was a PR opened in the meantime!